src/idl_gen_cpp.cpp

```diff
@@ -204,6 +204,19 @@ class CppGenerator : public BaseGenerator {
     if (num_includes) code_ += "";
   }
 
+  void GenExtraIncludes() {
+    if(parser_.opts.cpp_includes.empty()) {
+      return;
+    }
+    // non-const copy needed for std::strtok
+    std::string data = parser_.opts.cpp_includes;
+    for(char* pch = std::strtok(&data[0], ",");
+        pch != nullptr; pch = std::strtok(nullptr, ",")) {
+      code_ += "#include <" + std::string(pch) + ">";
+    }
+    code_ += "";
+  }
+
```

Comment:
`std::strtok` isn't necessary.
The `std::string::find_first_of` is better C++ alternative to `std::strtok`.
For example, see `CheckedError Parser::ParseEnumFromString` method.

Reply:
yeah, fails on windows, gotta update the patch.

Reply:
@vglavnyy fixed fyi.